application/sitebuilder/build.py

```diff
@@ -62,6 +76,35 @@ def enforce_static_mode():
             clear_up(build_dir)
 
 
+def build_and_upload_error_pages(application):
+    with application.app_context():
+        build_dir = new_build_dir(application)
+
+        # Inject static_mode=True into all Jinja render_template calls so that pages are automatically rendered in the
+        # correct mode.
+        @application.context_processor
+        def enforce_static_mode():
+            return dict(static_mode=True)
+
+        local_build = application.config["LOCAL_BUILD"]
+
+        build_error_pages(build_dir)
+
+        print("Deploy site (error pages) to S3: ", application.config["DEPLOY_SITE"])
```

Comment:
Remove print? Or alternatively maybe log this to the logger with a log level of 'info'?  (Unless that's what `print` does by default anyway?)

Reply:
This matches what we do at the moment in the current site builder function `do_it`, so it's consistent.

They probably _should_ be log calls though.

Reply:
I just tried replacing all these with calls to logger but they disappeared somewhere. Print is good for now.